cylc/flow/workflow_files.py

```diff
@@ -1700,11 +1699,13 @@ def get_run_dir_info(
 
 def detect_both_flow_and_suite(path):
     """Detects if both suite.rc and flow.cylc are in directory.
+
+    Permits flow.cylc to be a symlink.
```

Comment:
Is this desirable?

Reply:
@oliver-sanders, I believe you said we should not stop users from manually making a symlink.

Reply:
Shouldn't we check that the symlink points to the right suite.rc, and raise an error if not?

Reply:
If there is a flow.cylc there it will use that, regardless of if there is a suite.rc file there. Do we want to forbid users from symlinking other places? I can't think of a reason to forbid, I am probably missing something! I think so long as it is a valid workflow (which will be picked up with other checks) then do we really care how they are sourcing the workflow, it is a bit messy but that is the user's choice? I can't think of a reason to forbid although I am possibly missing something!

Reply:
It sounds reasonable to me that users could have a flow.cylc that is a symlink to either its neighbour suite.rc or somewhere else. But if they have a flow.cylc that symlinks to somewhere else *and* a suite.rc in there, that sounds to me like it should fail in the same way as a normal flow.cylc file and a suite.rc in there.

Reply:
Resolving this as there is a use case (at the MO) for users to source workflow from outside the run directory.

Reply:
Sorry Mel, not sure if you saw my latest comment above https://github.com/cylc/cylc-flow/pull/4506#discussion_r750305478

Reply:
Ah I didn't see that, thanks @MetRonnie, mustn't have refreshed the page before posting.